book/types_of_data.md

```diff
@@ -149,31 +85,22 @@ This chart shows all durations currently supported:
 | `1hr`    | one hour                    |
 | `1day`   | one day                     |
 | `1wk`    | one week                    |
-| `1month` | one month (30 days)         |
-| `1yr`    | one year (365 days)         |
-| `1dec`   | one decade (10 \* 365 days) |
-
-## Ranges
```

Comment:
The changes here aren't as dramatic as they look; it's just that I've moved Filesizes up to be next to Durations.